File: python/sprint5/G/code.py
# ! change LOCAL to False before submitting !
# set LOCAL to True for local testing

LOCAL = True

if LOCAL:
    class Node:  
        def __init__(self, value, left=None, right=None):  
            self.value = value  
            self.right = right  
            self.left = left

def solution(root) -> int:
    #  Your code
    #  “ヽ(´▽｀)ノ”
    pass
 

def test():
    node1 = Node(5, None, None)
    node2 = Node(1, None, None)
    node3 = Node(-3, node2, node1)
    node4 = Node(2, None, None)
    node5 = Node(2, node4, node3)
    assert solution(node5) == 6

if __name__ == '__main__':
    test()

File: go/sprint5/G/code.go
package main


// <template>
type Node struct {  
	value  int  
	left   *Node  
	right  *Node  
}
// <template>


func Solution(root *Node) int {
    // Your code
    // “ヽ(´▽｀)ノ”
}

func test() {
	node1 := Node{5, nil, nil}
    node2 := Node{1, nil, nil}
    node3 := Node{-3, &node2, &node1}
    node4 := Node{2, nil, nil}
    node5 := Node{2, &node4, &node3}
	if Solution(&node5) != 6 {
		panic("WA")
	}
}